### DISCUSSION TITLE
Entries field type: add the ability to filter related entries by a field value

### DISCUSSION THREAD

**Discussion by watershed** [craftcms/cms#9007] [Ideas]
Tags and categories are great for creating subsets of entries but they’re not curated. You can order them by various criteria, but sometimes one wants to create a curated subset of entries that relate to a specific field value such as a tag.

## A tag example

One can do something like create a Matrix field with a block that has: 

* A `Tag` field type from which to select a single tag
* A related `Entries` field type from which to select multiple entries

…and in so doing allow authors to build a group of curated subset blocks.

But there is no way I know of to have the set of `Entries` offered up in edit mode be filtered by the chosen constraint. The author would have to consult front end tag page output to verify that the entries they are choosing are indeed tagged with the desired association.

Perhaps there could be a way for authors to impose a filter during `Entries` search, where it is left up to the author to do that. And perhaps there could be a way to impose a specific constraint in field config.

**Comment by brandonkelly** [comment]
This will be possible in Craft 4. Basically the same request as #4299 (in reality we will be making the custom filters much more robust than just an entry type selection).

In the meantime you can add your own filtering logic via a module. See https://github.com/craftcms/cms/issues/4299#issuecomment-615192553 for an example.
